python/tvm/script/parser/tir/parser.py

```diff
@@ -411,6 +412,10 @@ def visit_expr_stmt(self: Parser, node: doc.Expr) -> None:
     if isinstance(res, Frame):
         res.add_callback(partial(res.__exit__, None, None, None))
         res.__enter__()
+    elif isinstance(res, PrimExpr):
+        T.evaluate(res)
+    elif isinstance(res, (int, bool)):
+        T.evaluate(tvm.tir.const(res))
```

Comment:
hey i was wondering if we want to explicitly print `T.evaluate(0)` vs `0`? The former one might look a bit clearer from my perspective

Reply:
I was going back and forth on it, and I like the idea of printing `T.evaluate()` except in the case of CallNode.  The more aggressive sugaring would be to render `tir::Evaluate(0)` as `pass`, which would be even clearer to read from a casual Python reader.

Reply:
This is a good idea!